### DISCUSSION TITLE
Straight Arduino Pro Micro

### DISCUSSION THREAD

**Discussion by willio** [alf45tar/PedalinoMini#296] [Q&A] [answered]
Hi, can this be done with only just an Arduino Pro Micro board without the ESP32? I'm thinking of making a 2 or 3 button switch controller without any display to keep things super simple.

**Comment by alf45tar** [comment, marked as answer]
Yes, it is possible but it is a different project. If you look around there are a lot of Arduino projects sending MIDI messages on button press.
